package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
     "build": "webpack",
     "server": "live-server",
```

Comment:
@ctong5 
@esme 

What is live-server being used for if you are also using express? If there is a valid use-case for live-server, I'd recommend adding it to the dev-dependencies, as it currently isn't listed. However, I'm assuming that this is just serving up static assets that express would otherwise be doing--what is the rationale on having two separate servers for these functions?